src/mono/wasm/runtime/loader/logging.ts

```diff
@@ -94,15 +94,17 @@ export function setup_proxy_console(id: string, console: Console, origin: string
     const consoleUrl = `${origin}/console`.replace("https://", "wss://").replace("http://", "ws://");
 
     consoleWebSocket = new WebSocket(consoleUrl);
-    consoleWebSocket.addEventListener("open", () => {
+    consoleWebSocket.onopen = () => {
```

Comment:
What is the reason for this change? `addEventListener` is generally preferred.

Reply:
during mono_exit we want to un-install the handler, but we don't have the original. This helps. Is there better way ?

Reply:
Let's not use lambdas if need to remove them

Reply:
done